components/esp_jpeg/README.md

```diff
@@ -0,0 +1,45 @@
+# JPEG Decoder: TJpgDec - Tiny JPEG Decompressor
+
+TJpgDec is a generic JPEG image decompressor module that highly optimized for small embedded systems. It works with very low memory consumption.
+
+Some microcontrollers have tjpg decoder in ROM, it is used, if there is. [^1] Using ROM code can be disabled in menuconfig.
+
+[^1]: **_NOTE:_** When there is used ROM decoder, there cannot be changed configuration. 
```

Comment:
```suggestion
[^1]: **_NOTE:_** When the ROM decoder is used, the configuration can't be changed. The configuration is fixed to **TODO** 
```

Please add more information about the fixed ROM configuration. + Information about how much program memory is saved by using the ROM version